### PR TITLE
[MIG] event_registration_mass_mailing: Migration to 12.0

### PR DESCRIPTION
**Pull request by chienandalu**
- We just keep features that aren't in Odoo already

cc @Tecnativa TT24616

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]
Functionally tested in runbot :+1: 

Please review travis :red_circle:

**Comment by rafaelbn** [comment]
```
2020-07-14 13:49:03,237 6516 INFO openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: test_add_to_mail_list (odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard.TestEventRegistrationMailListWizard)
2020-07-14 13:49:03,243 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: FAIL
2020-07-14 13:49:03,245 6516 INFO openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: ======================================================================
2020-07-14 13:49:03,245 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: FAIL: test_add_to_mail_list (odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard.TestEventRegistrationMailListWizard)
2020-07-14 13:49:03,245 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: Traceback (most recent call last):
2020-07-14 13:49:03,246 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: `   File "/home/travis/build/OCA/event/event_registration_mass_mailing/tests/test_event_registration_mail_list_wizard.py", line 45, in test_add_to_mail_list
2020-07-14 13:49:03,246 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: `     self.assertEqual(self.mail_list.contact_nbr, 2)
2020-07-14 13:49:03,246 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: ` AssertionError: 1 != 2
2020-07-14 13:49:03,246 6516 INFO openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: Ran 1 test in 0.125s
2020-07-14 13:49:03,246 6516 ERROR openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard: FAILED
2020-07-14 13:49:03,246 6516 INFO openerp_test odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard:  (failures=1)
2020-07-14 13:49:03,246 6516 ERROR openerp_test odoo.modules.module: Module event_registration_mass_mailing: 1 failures, 0 errors
2020-07-14 13:49:03,850 6516 INFO openerp_test odoo.modules.registry: module event_registration_partner_unique: creating or updating database tables
```
```

----------
FAIL
FAIL: test_add_to_mail_list (odoo.addons.event_registration_mass_mailing.tests.test_event_registration_mail_list_wizard.TestEventRegistrationMailListWizard)
Traceback (most recent call last):
`   File "/home/travis/build/OCA/event/event_registration_mass_mailing/tests/test_event_registration_mail_list_wizard.py", line 45, in test_add_to_mail_list
`     self.assertEqual(self.mail_list.contact_nbr, 2)
` AssertionError: 1 != 2
FAILED
Module event_registration_mass_mailing: 1 failures, 0 errors
At least one test failed when loading the modules.
Message not found: 'Modules loaded.'
----------
FAIL Command exited with code 1
FAIL Found 10 lines with errors
Module test summary
FAIL crm_lead_to_event_registration,event_contact,event_email_reminder,event_mail,event_project,event_registration_cancel_reason,event_registration_mass_mailing,event_registration_multi_qty,event_registration_partner_unique,event_sale_registration_multi_qty,event_sale_session,event_session,event_session_registration_multi_qty,event_track_location_overlap,partner_event,website_event_excerpt_img,website_event_filter_organizer,website_event_filter_selector,website_event_questions_free_text,website_event_questions_template,website_event_require_login,website_event_share,website_event_snippet_calendar,website_event_type_description
```

File: event_registration_mass_mailing/README.rst [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/event-12-0/event-12-0-event_registration_mass_mailing
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/199/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
 This module adds actions to send mass mailings and allows to add to mailing
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/README.rst]: `send mass mailings` action is still there? Shouldn't be removed as overlapping?

File: event_registration_mass_mailing/__manifest__.py [hunk 2]
```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Antiun Ingenieria S.L. - Javier Iniesta
 # Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2020 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
 {
-    'name': "Mass mailing from events",
+    'name': "Event mass mailing helper",
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/__manifest__.py]: Be more specific

File: event_registration_mass_mailing/views/event_registration.xml [hunk 3]
```diff
@@ -1,72 +1,36 @@
...
-    target="new"
-    key2="client_action_multi"
-    id="action_event_registration_mass_mail"
-    context="{
-        'default_composition_mode': 'mass_mail',
-        'default_partner_to': '${object.partner_id.id or \'\'}',
-        'default_use_template': True,
-        'default_template_id': ref('email_template_event_registration')
-    }"/>
+    <act_window name="Mass mailing to partners"
```
Comment (pedrobaeza) [inline event_registration_mass_mailing/views/event_registration.xml]: Why the need of this? I don't think this should be migrated. You already have mailing schedule at events, and mass_mailing with event registrations source.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: event_registration_mass_mailing/views/event_registration.xml (hunk 3 above)
Comment (chienandalu) [reply event_registration_mass_mailing/views/event_registration.xml]: Ok. Let's get rid of it

**Comment by chienandalu** [comment]
@pedrobaeza Changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**